sentinel-adapter/sentinel-apache-dubbo-adapter/README.md

```diff
@@ -65,4 +65,10 @@ flow control, degrade or system load protection. You can implement your own `Dub
 and then register to `DubboFallbackRegistry`. If no fallback is configured, Sentinel will wrap the `BlockException`
 then directly throw it out.
 
-Besides, we can also leverage [Dubbo mock mechanism](http://dubbo.apache.org/en-us/docs/user/demos/local-mock.html) to provide fallback implementation of degraded Dubbo services.
\ No newline at end of file
+Besides, we can also leverage [Dubbo mock mechanism](http://dubbo.apache.org/en-us/docs/user/demos/local-mock.html) to provide fallback implementation of degraded Dubbo services.
+
+## Global dubbo provider origin parse
+
+Sentinel Dubbo Adapter supports global origin parse for provider.
+You can implement your own `DubboOriginParser` interface
+and then register to `DubboAdapterGlobalConfig`. If no originParse is configured, Sentinel will user dubbo url property application.
```

Comment:
`originParse` seems a typo, adding `Since 1.8.0` can help user who use old verison. 
Maybe the document can refer to #1572

Reply:
Ok, this piece is really my incomplete consideration, thank you for correcting, and it has been fixed